packages/e2e/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "e2e",
-  "version": "3.1.2",
+  "version": "5.0.0",
```

Comment:
why are we going to v5 ?

Reply:
done by mistake while playing with Yarn commands :)